server/internal/infrastructure/db/sqlite/migration/20250618123533_tx_graph.up.sql

```diff
@@ -0,0 +1,12 @@
+ALTER TABLE tx DROP COLUMN tree_level;
+ALTER TABLE tx DROP COLUMN parent_txid;
+ALTER TABLE tx DROP COLUMN is_leaf;
+
+ALTER TABLE tx ADD COLUMN children TEXT;
```

Comment:
@sekulicd this children is a map[uint32]string that (at the moment) can contain max 2 entries. Here we're storing its json serialization in sqlite. Do you think it's ok? Do you suggest something else?